Header header

uint8 radar_num
uint8 num_objects
float32[32] accel_x
float32[32] vel_x
float32[32] pos_x
float32[32] pos_y
float32[32] exist_prob
bool[32] valid
